tests/cpp/test_combine_mul_sum.cpp

```diff
@@ -38,6 +39,11 @@
 namespace nvfuser {
 
 class CombineMulSumAsMmaTest : public NVFuserTest {
+ protected:
+  CombineMulSumAsMmaTest() {
+    disable_options_ptr->getCurOptions().set(DisableOption::MatmulExprEval);
```

Comment:
```suggestion
    DisableOptionsGuard::getCurOptions().set(DisableOption::MatmulExprEval);
```
because it's a static method. It's technically possible to use it on an instance, but that gives readers the wrong impression.